modules/nf-core/nanoq/tests/main.nf.test.snap

```diff
@@ -0,0 +1,307 @@
+                            "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                        ]
+                    ]
+                ],
+                "2": [
+                    "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                ],
+                "reads": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        [
+                            "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe",
+                            "test_nanoq.fastq.gz:md5,7567d853ada6ac142332619d0b541d76",
+                            "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                        ]
+                    ]
+                ],
+                "stats": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe"
+                    ]
+                ],
+                "versions": [
+                    "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                ]
+            }
+        ],
+        "meta": {
+            "nf-test": "0.8.4",
+            "nextflow": "23.04.1"
+        },
+        "timestamp": "2024-06-28T10:54:00.467634"
+    },
+    "sarscov2 - nanopore_uncompressed_bz2": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe"
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        [
+                            "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe",
+                            "test_nanoq.fastq.bz2:md5,b53cf14fd4eb5b16c459c41f03cc8a4b",
+                            "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                        ]
+                    ]
+                ],
+                "2": [
+                    "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
+                ],
+                "reads": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": true
+                        },
+                        [
+                            "test.stats:md5,5ab32af3352dfeca8268e10edf6e4dbe",
+                            "test_nanoq.fastq.bz2:md5,b53cf14fd4eb5b16c459c41f03cc8a4b",
+                            "versions.yml:md5,7a40efe417ff7dbb9e91e9c1629a04e6"
```

Comment:
This looks like your expression for the output channel is capturing all these.

Reply:
Updated.